app/services/web_mentions/web_mention_handler.rb

```diff
@@ -0,0 +1,37 @@
+module WebMentions
+  class WebMentionHandler
+    def initialize(canonical_url, article_url = nil)
+      @canonical_url = canonical_url
+      @article_url = article_url
+      @webmention_url = ""
+    end
+
+    def self.call(*args)
+      new(*args).call
+    end
+
+    def call
+      accepts_webmention? ? send_webmention(@webmention_url) : Rails.logger.info("#{@canonical_url} doesn't support Webmentions")
+    end
+
+    def accepts_webmention?
```

Comment:
I find this a bit misleading. The name suggests a boolean check, not something that actually sets `@webmention_url` as a side effect.

Reply:
Good point @citizen428, wouldn't this be better? 
```
def accepts_webmention?
  webmention_url.blank? ? false : true
end

def webmention_url
  # add webmention_url setting logic here
end
```

Reply:
Instead of

```ruby
def accepts_webmention?
  webmention_url.blank? ? false : true
end
```

we can just do 

```ruby
def accepts_webmention?
  webmention_url.present?
end
```

The predicate method already returns a boolean and `present? == !blank?`